libs/base/math.ts

```diff
@@ -69,13 +84,21 @@ namespace Math {
     }
 
     /**
-     * Returns the cosine of an input angle. This is an 8-bit approximation. 
+     * Returns the cosine of an input angle. This is an 8-bit approximation.
      * @param theta input angle from 0-255
      */
     //% help=math/icos weight=10 advanced=true blockGap=8
     export function icos(theta: number) {
         return isin(theta + 16384);
     }
+
+    //% shim=TD_ID
+    //% block="$MEMBER"
+    //% constantShim
+    //% weight=0
+    export function _constant(MEMBER: number): number {
```

Comment:
Does something special happen by using MEMBER? Or is it the `constantShim` that correctly populates all of the constants?

Reply:
yeah, this is me tricking the decompiler into using the right name for the field this generates. i would have added a comment but then it would have shown up in the tooltip for the block